assets/src/apps/authoring/components/Flowchart/toolbar/FlowchartHeaderNav.tsx

```diff
@@ -363,11 +372,14 @@ export const FlowchartHeaderNav: React.FC<HeaderNavProps> = () => {
                   <Tooltip id="button-tooltip" style={{ fontSize: '12px' }}>
                     <strong>Paste Component</strong>
                     {hasQuestion && <div>Only one question component per screen is allowed</div>}
+                    {isLessonEndScreen && (
```

Comment:
also doing this makes me feel funny because it's a bit specific. maybe I'd prefer having some kind of `canAddQuestions` variable instead which right now would be `!isLessonEndScreen` but could change later to have other criteria;

separately from that however, I notice there is some sort of screen validation thing that already has some rules and reasons in it, can that be enhanced instead?

Reply:
I thought of renaming the variable but the `'Question/interaction components cannot be added to the last screen.'` validation message was too specific to the end of lesson use case.